### ISSUE TITLE
🐛 Bug Report: The commits list is rendered with the loader

### ISSUE THREAD

**Issue by abrl91** [amplication/amplication#4199]
### What happened?

The commits list is rendered with the loader

https://user-images.githubusercontent.com/39680385/196733731-94022b76-51b0-4f86-88b7-9d7883845ce6.mov

### What you expected to happen

I expect to see the loader, and only after a few milliseconds the list should be rendered on the **top** of the sidebar 

### How to reproduce

show the video

**Comment by TheLearneer** [comment]
Hi @abrl91 👋 
I would love to work on this issue 🙇

**Comment by DivyanshVinayak23** [comment]
Hi @abrl91 Can I work on this?

**Comment by abrl91** [comment]
> Hi @abrl91 Can I work on this?

sorry, @TheLearneer asked first. I will open another issue like that today and if you want I can assign you immediately

**Comment by abrl91** [comment]
@DivyanshVinayak23 please comment on this issue: https://github.com/amplication/amplication/issues/4211 so I would be able to assign you

**Comment by abrl91** [comment]
Fixed by https://github.com/amplication/amplication/pull/4212
